libsql-storage/proto/storage.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
```

Comment:
nit: I prefer to go with `MessageRequest` and `MessageResponse` this seems to be the style google uses. I find resp and res to be really easy to confuse.